src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -446,6 +447,7 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             await self._handle_user_image_frame(frame)
         elif isinstance(frame, BotStoppedSpeakingFrame):
             await self.push_aggregation()
+            await self.push_frame(frame, direction)
```

Comment:
It's weird this fixes the problem because the assistant aggregator goes after the output transport and the STT mute filter is way up in the pipeline. So, I'm not sure how could that affect the STT mute filter. How does the pipeline look like?